src/test/utils.ts

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
Is there a better name than `utils.ts`? Perhaps `setupAndTeardown.ts`, `harnessFunctions.ts`, `databaseSetup.ts`? Something like that.